app/src/main/java/org/mifos/mobilebanking/ui/views/RecentTransactionsView.java

```diff
@@ -42,6 +42,8 @@
 
     void showMessage(String message);
 
+    void showToastMessage(String message);
```

Comment:
@MigDinny I meant to remove this and write code show message in `showMessage()`

Reply:
I didn't want to do it because it would probably ruin other showMessage() callings where a snackbar is the expected behavior.

I will close this PR and create a new one with a new branch